jobs/rabbitmq-server/templates/plugins.sh.erb

```diff
@@ -7,6 +7,9 @@ write_log() {
 }
 
 (
+  <% if_p("rabbitmq-server.additional_plugin_dir") do |dir| %>
+  export RABBITMQ_PLUGINS_DIR=/var/vcap/packages/rabbitmq-server/plugins/:<%= dir %>
+  <% end %>
```

Comment:
I believe that we can just set the variable here instead of exporting it.

This is because the plugins enable command is run soon after in the same process.

Reply:
Fair enough @teancom , thanks !